src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Feedback/FeedbackFileLoggerProvider.cs

```diff
@@ -2,15 +2,18 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.ComponentModel.Composition;
 using Microsoft.Extensions.Logging;
 
 namespace Microsoft.VisualStudio.LanguageServerClient.Razor.Feedback
 {
+    [Export(typeof(ILoggerProvider))]
```

Comment:
This is too generic of an interface to export

Reply:
Scratch that, as we're using the factory, we don't need to export it in the first place. 👍